ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -34,6 +35,9 @@ def append_tax_details_into_item_lines(invoice_id: str, item_lines: list, conver
             tax_percent = 0.0
             tax_amount = 0.0
 
+        item["amount"] = abs(item["amount"]) - tax_amount if is_tax_included else item["amount"]
```

Comment:
* Can tax_amount be negative in return scenarios, similar to amount?
* Does tax_amount include qty or is it for one item?
* What's the reasoning behind moving the multiplication by qty here instead of the template?

Reply:
tax_amount cannot be negative in case of return and it includes the qty of the item also for the item["amount"].
PriceAmount(BT-146) = BaseAmount(BT-148) - Allowance Amount (BT-147) when BaseAmount is provided in XML,
thats why it included the qty in the xml to add the total discount to the item amount to calculate BaseAmount.